snapcraft/internal/repo/_platform.py

```diff
@@ -34,7 +35,7 @@ def _is_deb_based(distro):
 
 
 def _get_repo_for_platform():
-    distro = _linux_distribution()[0]
+    distro = _linux_distribution()[0].lower()
```

Comment:
Doing a `lower` here will fail all the other matches it needs to do. I would suggesst against doing a lower here unless everything is lowered in `_DEB_BASED_PLATFORM` but that makes the latter less strict so I am more inclined on adding all the matches you need.